pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "Cython~=0.29",
+    "Cython",
```

Comment:
Do we want to relax this? Like this, or still add some form of upper pin? cc @mwtoews

Reply:
Yes, it can be relaxed.